conf.py

```diff
@@ -16,11 +14,10 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Common Package Specification'
-copyright = u'2023, Matthew Woehlke'
+project = 'Common Package Specification'
+copyright = '2023, Matthew Woehlke'
```

Comment:
If we're touching this line anyway, might as well fix it too; it should be 2024 now...

Reply:
You can remove technically remove the year. It just keeps copyright in perpetuity until handed over/passed on.

Reply:
Updated the year and removed the `u` in my merge commit.